classes/output/mobile.php

```diff
@@ -426,7 +426,7 @@ public static function view_discussion($args) {
                             $reply->imgtype = true;
                             array_push($reply->images, $fileobj);
                             break;
-                        case 'image/jpg':
+                        case 'image/jpeg':
```

Comment:
What if the image type is jpg?

Reply:
I uploaded type jpg and looks likes the mimetype is always set to jpeg, I thought of the same thing, but could not see that coming up during my local testing will add it to testing steps on the ticket just in case